flaml/automl/ml.py

```diff
@@ -678,25 +692,28 @@ def compute_estimator(
 
 
 def train_estimator(
-    config_dic,
+    config_dic: dict,
     X_train,
     y_train,
-    task,
-    estimator_name,
-    n_jobs=1,
-    estimator_class=None,
+    task: str,
+    estimator_name: str,
+    n_jobs: Optional[int] = 1,

```

Comment:
Does the comment in 628 apply to here as well?

Reply:
It does. It's just a reminder reported from class `SKLearnEstimator`. Not sure what should I do: should I remove the inline comment altogether or just copy it once again here?

Reply:
> It does. It's just a reminder reported from class `SKLearnEstimator`. Not sure what should I do: should I remove the inline comment altogether or just copy it once again here?

You can copy it again here.

Reply:
> Also, there's something weird going on - I am not able to reproduce the pipeline error locally. Seems that PR #918 fails in the same test, do you have any idea why?

@int-chaos is investigating it. Most likely some dependencies changed versions.

Reply:
> > It does. It's just a reminder reported from class `SKLearnEstimator`. Not sure what should I do: should I remove the inline comment altogether or just copy it once again here?
> 
> You can copy it again here.

This has not been addressed.